services/resource-usage-tracker/tests/unit/with_dbs/test_collect_container_resource_usage_task__on_update_set.py

```diff
@@ -77,12 +77,14 @@ def random_promql_output_generator():
                 "container_label_com_docker_compose_oneoff": "False",
                 "container_label_com_docker_compose_project_working_dir": "/tmp/tmp_3seh6kp",
                 "container_label_com_docker_compose_version": "1.29.1",
-                "container_label_product_name": "osparc",
+                "container_label_io_simcore_runtime_product_name": "osparc",
                 "container_label_simcore_service_settings": '[{"name": "ports", "type": "int", "value": 8888}, {"name": "env", "type": "string", "value": ["DISPLAY=:0"]}, {"name": "env", "type": "string", "value": ["SYM_SERVER_HOSTNAME=sym-server_%service_uuid%"]}, {"name": "mount", "type": "object", "value": [{"ReadOnly": true, "Source": "/tmp/.X11-unix", "Target": "/tmp/.X11-unix", "Type": "bind"}]}, {"name": "constraints", "type": "string", "value": ["node.platform.os == linux"]}, {"name": "Resources", "type": "Resources", "value": {"Limits": {"NanoCPUs": 4000000000, "MemoryBytes": 17179869184}, "Reservations": {"NanoCPUs": 100000000, "MemoryBytes": 536870912, "GenericResources": [{"DiscreteResourceSpec": {"Kind": "VRAM", "Value": 1}}]}}}]',
-                "container_label_simcore_user_agent": "puppeteer",
-                "container_label_study_id": "46449cc3-7d83-4081-a44e-fc75a0c85f2c",
-                "container_label_user_id": "43820",
-                "container_label_uuid": "2b231c38-0ebc-5cc0-1234-1ffe573f54e9",
+                "container_label__io_simcore_runtime_imcore_user_agent": "puppeteer",
```

Comment:
Warning this double underscore in `"container_label__io_simcore_runtime_imcore_user_agent"` is correct?

Reply:
oh god, let's please avoid double underscores at all costs, the last `$` killed our invitation service and it was only a single char 🤣

Reply:
Yes my mistake, this variable is not used by the resource tracker, but I will correct it so its right, thanks